test/e2e/specs/specs-playwright/wp-signup__free.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * @group calypso-release
+ * @group quarantined
```

Comment:
I'm merging my updated PR today (unless there's big issues with it!) so I think you can leave this out for now.

Unless I totally make these specs worse, and then we can put them back in there 🙃

Reply:
I was thinking we can run your updated tests in the quarantined build while we ensure it is stable, hence this change. Let's take the other approach and demote it if necessary instead.